Library/Formula/pmd.rb

```diff
@@ -25,6 +25,7 @@ def caveats; <<-EOS.undent
   end
 
   test do
+    ENV["_JAVA_OPTIONS"] = "-Duser.home=#{ENV["HOME"]}”
```

Comment:
this would need to happen in the `install` method, not here, but I'd rather not work around this in the formula. Thanks for the update though!